pkgs/tools/misc/togglesg-download/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, buildPythonApplication, makeWrapper, ffmpeg }:
+
+buildPythonApplication rec {
+
+  name = "togglesg-download-git-${version}";
+  version = "2016-02-08";
+
+  src = fetchFromGitHub {
+    owner = "0x776b7364";
+    repo = "toggle.sg-download";
+    rev = "5cac3ec039d67ad29240b2fa850a8db595264e3d";
+    sha256 = "0pqw73aa5b18d5ws4zj6gcmzap6ag526jrylqq80m0yyh9yxw5hs";
+  };
+
+  buildInputs = [ makeWrapper ];
```

Comment:
I thought ```nativeBuildInputs``` were only used when cross compiling. What difference would it make here?